opentimelineio/adapters/cmx_3600.py

```diff
@@ -591,6 +641,28 @@ def write_to_string(input_otio):
         if url:
             lines.append("* FROM CLIP: {}".format(url))
 
+        cdl = clip.metadata.get('cdl')
+        if cdl:
+            asc_sop = cdl.get('asc_sop')
+            asc_sat = cdl.get('asc_sat')
+            if asc_sop:
```

Comment:
Is it valid to have a CDL with SAT but not SOP data? Or the other way?  That would be an argument for pushing for a schema.  This code looks good to me for now though.

Reply:
I made them separate like that in case you had one and not the other. Some of the reference I read implied that could happen, but I haven't seen it in practice. There was also some documentation that implied that you could stack multiple SOPs or SATs on a single clip (in an FCP XML not an EDL). I think we should defer support for that until we do the FCP XML side of this, which is also when we will likely want to treat this as a video effect instead of metadata.